python-avocado.spec

```diff
@@ -172,6 +172,9 @@ popd
 pushd optional_plugins/runner_docker
 %{__python} setup.py develop --user
 popd
+pushd optional_plugins/resultsdb
+%{__python} setup.py develop --user
+popd
```

Comment:
I see, basically the `build`, `install` and `check` should setup the same components (with some additions in `install`). Yes, `resultsdb` was actually missing here....